### PR TITLE
Minor changes to bug template

### PR DESCRIPTION
**Pull request by yodaluca23**
Minor changes to the bug template, including clarification of where the title goes as I have seen lots of people change the name of their issue after the original creation.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ _Make sure you complete the template. Otherwise, it will be closed without furth
 _(Please attach videos or screenshots if possible)_
 
 ### Crash log (if possible)
-_(The crash log can be found in the Settings app > Privacy > Analytics & Improvements > Analytics Data > Youtube...)_
+_(The crash log can be found in the iPhone Settings App > Privacy > Analytics & Improvements > Analytics Data > Youtube...)_
```
Comment (PoomSmart) [inline .github/ISSUE_TEMPLATE/bug.md:27]: This can be iPad or iPod also.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 about: Make sure you complete the template. Otherwise, it will be closed without further explanation!
-title: "[Bug]"
+title: "[Bug] (Replace this with your title)"
 labels: bug
 assignees: ''
 
 ---
 
-## ⚠️ Have you read the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ)? Make sure you visit the FAQ page first!!! ⚠️
+## ⚠️ Have you read the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ)? Make sure you visit the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ) page first!!! ⚠️
```
Comment (PoomSmart) [inline .github/ISSUE_TEMPLATE/bug.md:10]: Is it necessary to add the link to the second appearing term?

Comment (yodaluca23) [reply .github/ISSUE_TEMPLATE/bug.md:10]: I don't know, I thought that I might as well...

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (yodaluca23) [reply .github/ISSUE_TEMPLATE/bug.md]: Fixed

**Pull request merged**